backend/de.metas.material/planning/src/main/java/de/metas/material/planning/pporder/ComputeQtyRequiredRequest.java

```diff
@@ -23,18 +23,34 @@
 package de.metas.material.planning.pporder;
 
 import de.metas.quantity.Quantity;
+import de.metas.util.Check;
 import lombok.Builder;
 import lombok.NonNull;
 import lombok.Value;
 import org.eevolution.api.ProductBOMLineId;
 
+import javax.annotation.Nullable;
+
 @Value
 @Builder
 public class ComputeQtyRequiredRequest
 {
-	@NonNull
+	public ComputeQtyRequiredRequest(@Nullable final Quantity finishedGoodQty,
+			@Nullable final Quantity issuedQty,
+			@NonNull final ProductBOMLineId productBOMLineId)
+	{
+		Check.assume(finishedGoodQty != null || issuedQty != null, "at least finishedGoodQty={} or issuedQty={} shall be not null", finishedGoodQty, issuedQty);
```

Comment:
i think we need `^` (XOR) between those, i.e. one and only one of those qtys shall be provided

Reply:
makes sense.